src/main/kotlin/org/simplemc/simplehealthbars2/Extensions.kt

```diff
@@ -3,18 +3,21 @@ package org.simplemc.simplehealthbars2
 import org.bukkit.attribute.Attribute
 import org.bukkit.entity.LivingEntity
 import org.bukkit.entity.Player
+import kotlin.math.max
 
-fun LivingEntity.getDamagedHealth(damage: Double): Double = Math.max(health - damage, 0.0)
+fun LivingEntity.getDamagedHealth(damage: Double): Double = max(health - damage, 0.0)
 
 fun LivingEntity.getDamagedHealthRatio(damage: Double): Double =
     getDamagedHealth(damage) / checkNotNull(getAttribute(Attribute.GENERIC_MAX_HEALTH)).value
 
 fun LivingEntity.getCustomDisplayName(): String = when {
     this is Player -> displayName
     isCustomNameVisible -> checkNotNull(customName)
-    else -> type.name.toLowerCase().split('_').joinToString(separator = " ", transform = String::capitalize)
+    else -> type.name.lowercase().split('_').joinToString(separator = " ", transform = String::titlecaseFirstChar)
 }
 
+fun String.titlecaseFirstChar() = replaceFirstChar(Char::titlecase)
```

Comment:
Did they remove / deprecate `capitalize` in kotlin?! That's definitely going to effect some of my projects lol

Reply:
Deprecated in 1.5: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/capitalize.html

...I _think_ they did it because it wasn't actually clear what the behavior was by name alone.